src/tools/default_flags_table/reco_flags.py

```diff
@@ -74,7 +74,100 @@
+    ('BEMC:EcalBarrelImagingRawHits:scaleResponse',             '1.0',                             ''),
+    ('BEMC:EcalBarrelImagingRawHits:signalSumFields',           '',                                ''),
+    ('BEMC:EcalBarrelImagingRawHits:fieldRefNumbers',           '',                                ''),
+    ('BEMC:EcalBarrelImagingRawHits:geoServiceName',            'ActsGeometryProvider',            ''),
+    ('BEMC:EcalBarrelImagingRawHits:readoutClass',              'pixel',                           ''),
+
+    ('BEMC:EcalBarrelScFiRawHits:input_tag',                     'EcalBarrelScFiHits',             'Name of input collection to use'),
+    ('BEMC:EcalBarrelScFiRawHits:energyResolutions',             '',                               ''),
+    ('BEMC:EcalBarrelScFiRawHits:timeResolution',                '0*ns',                           ''),
+    ('BEMC:EcalBarrelScFiRawHits:capacityADC',                   '16192',                          ''),
+    ('BEMC:EcalBarrelScFiRawHits:dynamicRangeADC',               '750*MeV',                        ''),
+    ('BEMC:EcalBarrelScFiRawHits:pedestalMean',                  '20',                             ''),
+    ('BEMC:EcalBarrelScFiRawHits:pedestalSigma',                 '0.3',                            ''),
+    ('BEMC:EcalBarrelScFiRawHits:resolutionTDC',                 '10*picosecond',                  ''),
+    ('BEMC:EcalBarrelScFiRawHits:scaleResponse',                 '1.0',                            ''),
+    ('BEMC:EcalBarrelScFiRawHits:signalSumFields',               '',                               ''),
+    ('BEMC:EcalBarrelScFiRawHits:fieldRefNumbers',               '',                               ''),
+    ('BEMC:EcalBarrelScFiRawHits:geoServiceName',                'ActsGeometryProvider',           ''),
+    ('BEMC:EcalBarrelScFiRawHits:readoutClass',                  'light_guide',                    ''),
+
+    ('BEMC:EcalBarrelImagingRecHits:input_tag',                 'EcalBarrelHits',                  'Name of input collection to use'),
+    ('BEMC:EcalBarrelImagingRecHits:layerField',                'layer',                           ''),
+    ('BEMC:EcalBarrelImagingRecHits:sectorField',               'module',                          ''),
+    ('BEMC:EcalBarrelImagingRecHits:capacityADC',               '8096',                            ''),
+    ('BEMC:EcalBarrelImagingRecHits:pedestalMean',              '400',                             ''),
+    ('BEMC:EcalBarrelImagingRecHits:dynamicRangeADC',           '100*MeV',                         ''),
+    ('BEMC:EcalBarrelImagingRecHits:pedSigmaADC',               '3.2',                             ''),
+    ('BEMC:EcalBarrelImagingRecHits:thresholdADC',              '3.0',                             ''),
+    ('BEMC:EcalBarrelImagingRecHits:samplingFraction',          '0.10262666247845109',             ''),
```

Comment:
0.1026 is actually the value for ScFi, it is much smaller for imaging layers (0.005).
But since this value is likely changed with the geometry update, I assume it will be dynamically read from the calibration files.

Reply:
Not sure, if I understand, Chao. The `BEMC:EcalBarrelImagingRecHits:samplingFraction` should be set to ~0.5% (if not assigned dynamically from the detector) and `BEMC:EcalBarrelSciFiRecHits:samplingFraction` should be ~12.5% (if not assigned dynamically from the detector).

Reply:
Yes, I meant this value (for imaging layers) should be around 0.5%. here 10.2% is incorrect because it was for ScFi layers. Also, as you mentioned, the current ScFi sampling fraction is 12.5% (changed from 10.2%) because we had a recent geometry update, that's why it is important to read these values dynamically.

Reply:
All good! I understand now what you meant. And I agree, the dynamic assignment is important!

Reply:
I'm not sure if I am following the lingo. When you are talking about "read dynamically" do you mean the number is set in the simulation (via calibration files) and that the reconstruction should also read from those calibration files rather than set them here? That's fine and makes sense, but since we are not currently set up to do that, we will need to match the values here. If I'm following you, then this means I need to set:

BEMC:EcalBarrelImagingRecHits:samplingFraction = 0.005
BEMC:EcalBarrelScFiRecHits:samplingFraction = 0.125

I'll do this now and you can let me know if I'm misunderstanding.

Reply:
Yes, this is what we meant. Sorry for not being clear. I agree, let's hardcode it now to these values.